var/spack/repos/builtin/packages/hpx/package.py

```diff
@@ -156,7 +156,8 @@ def cmake_args(self):
             '-DBOOST_ROOT={0}'.format(spec['boost'].prefix),
             '-DHWLOC_ROOT={0}'.format(spec['hwloc'].prefix),
             '-DHPX_WITH_BOOST_ALL_DYNAMIC_LINK=ON',
-            '-DBUILD_SHARED_LIBS=ON'
+            '-DBUILD_SHARED_LIBS=ON',
+            '-DHPX_DATASTRUCTURES_WITH_ADAPT_STD_TUPLE=OFF'
```

Comment:
Just to be clear, this turns the option *off* which is understandable since the default is on, but inconsistent with the title of the PR. Is this intentional? I would also make this a (spack) configuration option instead of turning it off completely (perhaps `adapt_std_tuple`). Does having it enabled prevent you from building your application? Is this related to https://github.com/STEllAR-GROUP/hpx/issues/4371?

Reply:
@msimberg,
Thanks for catching the mistake, I just fix the title.
Yes, this triggered the [4371](https://github.com/STEllAR-GROUP/hpx/issues/4371). 
So I guess, it would be the best, if I retract this PR and just wait for the HPX issue to get resolved?

Reply:
Either way. The option can be useful to have in the spack package but then it really has to be an option, not forced to off.